js/main.js

```diff
@@ -243,9 +256,16 @@ function clearDeck(dom) {
   }
 }
 
-if (Object.keys(data.myDeck).length !== 0) {
-  $deckImage.textContent = '';
-  const $coverImage = document.createElement('img');
-  $coverImage.setAttribute('src', data.myDeck[deckCardIds[0]].img);
-  $deckImage.appendChild($coverImage);
+function toggleNoCards() {
+  if (Object.keys(data.myDeck).length === 0) {
+    for (let i = 0; i < $noCards.length; i++) {
+      $noCards[i].style.display = 'flex';
+      $deckImageCover.style.display = 'none';
+    }
+  } else {
+    for (let i = 0; i < $noCards.length; i++) {
+      $noCards[i].style.display = 'none';
+      $deckImageCover.style.display = 'flex';
```

Comment:
Fine but we want to avoid inline styling. It's fine for stuff like `display: none`, but when we start getting into `flex` and additional styles, we want to apply these via classes.